packages/users/src/vault/types.ts

```diff
@@ -0,0 +1,54 @@
+export enum VaultItemType {
+  PrivateKeyWithMnemonic = 'PrivateKeyWithMnemonic',
+}
+
+export enum VkVersion {
+  VkVersion1 = 'V1',
+}
+
+/**
+ * Object representing item stored or retrieved from the vault.
+ *
+ */
+export interface VaultItem {
+  itemType: VaultItemType;
+  value: string;
+}
+
+export enum VaultBackupType {
+  Password = 'password',
```

Comment:
This won't be an option going forward but up to you if you want to clean it up here or not.

Reply:
True, I'll clean it up.